projects/UIP.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: project
+title: UIP
+permalink: projects/UIP
+logo: "https://raw.githubusercontent.com/NIT-dgp/UIP-misc/master/artwork/UIP_logo_v4.png"
+description: A cross-platfrom desktop application for wallpapers.
```

Comment:
still description is same @nkprince007  ? shouldnt it be `for changing and scheduling wallpapers` ?